### DISCUSSION TITLE
Icons for custom search engines

### DISCUSSION THREAD

**Discussion by embyveddii** [Floorp-Projects/Floorp#681] [Feature Request]
I'm not sure why it's not a given, but.. yeah. Icons on custom search engines? That way they are distinguishable from each other and not a block of grey icons lol
![image](https://github.com/Floorp-Projects/Floorp/assets/59317664/0a136356-ff7a-4e34-88cd-81bd984b1b16)

**Comment by Ryosuke-Asano** [comment]
So... I recommended installing search engine from firefox

**Reply by embyveddii** [reply]
that seems silly though. If firefox doesn't let me add it for whatever reason (as it has in the past on the normal version) then I just have to live with a grey search icon?
It's up to you if you wanna bother adding it lol. But it should be easy to get the site's favicon, bookmarks already do it. It would be a QOL improvement.
